hosting/single/default

```diff
@@ -0,0 +1,94 @@
+server {
```

Comment:
could we name this file something with nginx in it, just make it clear its an nginx configuration file? like `nginx-default.conf` and then just name it as required when being passed into the build? Just thinking the directory is full of a few different config files and its probably good to make their purpose obvious without the need to look through them.